.gitignore

```diff
@@ -27,6 +27,5 @@ npm-debug.log
 npm-debug.log.*
 package-lock.json
 
-# WebStorm user-specific
-.idea/workspace.xml
-.idea/tasks.xml
+.idea
+.vscode
```

Comment:
Those directories are present in upstream.
Could you please remove them from the .gitignore ... this will simplify PR to upstream (if any one day... :))
Thanks

Reply:
Changed, although they should really not be adding those.